### PR TITLE
papr: changes related to f27 release

### PR DESCRIPTION
**Pull request by miabbott**
F27 has been released, so let's enable our CI on that image and switch
the containers to F27, too.

### CODE REVIEW

**Comment by miabbott** [comment]
In the spirit of spreading our review coverage around, @jlebon do you want to have peek?

File: .papr.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ cluster:
     - name: testnode
       distro: fedora/26/atomic
   container:
-    image: registry.fedoraproject.org/fedora:25
+    image: registry.fedoraproject.org/fedora:27
```
Comment (jlebon) [inline .papr.yml]: I'd just keep 26 here to make sure running from the f26 userspace is still OK. We're already testing 27 down there right?

Comment (miabbott) [reply .papr.yml]: In the same vein, does it make sense to switch the `centos/7/atomic` contexts to use a CentOS base image?

Comment (jlebon) [reply .papr.yml]: I'd say so yeah. More so than f26 vs f27. :)

**Comment by jlebon** [comment]
LGTM! 👍

**Pull request merged**